packages-ui/roosterjs-react/lib/ribbon/type/RibbonButton.ts

```diff
@@ -37,7 +37,7 @@ export default interface RibbonButton<T extends string> {
     onClick: (
         editor: IEditor,
         key: T,
-        strings: LocalizedStrings<T>,
+        strings: Partial<LocalizedStrings<T>> | undefined,
```

Comment:
same

Reply:
why allow undefined here?

Reply:
`create[X]Plugin` functions have the optional `strings?` parameter, which is then passed to all the other components and functions. `getLocalizedString` receives these optional objects, and it should be able to handle undefined cases 
https://github.com/microsoft/roosterjs/blob/f8198d2c220a2fc363de69e0a74afa4ab2daefb3/packages-ui/roosterjs-react/lib/common/utils/getLocalizedString.ts#L10-L24

Reply:
I see.